src/nomad_measurements/utils.py

```diff
@@ -106,3 +109,19 @@ def merge_sections(
                 logger.warning(warning)
             else:
                 print(warning)
+
+
+def set_quantity(value: Any=None, unit: str=None) -> Any:
```

Comment:
I would suggest changing the name to `get_quantity`

Reply:
Modifying it to `to_pint_quantity`